plutus-use-cases/test/Spec/Prism.hs

```diff
@@ -165,11 +165,11 @@ instance ContractModel PrismModel where
         case cmd of
             Revoke w  -> do
               issued <- use (isIssued w)
-              when (issued == Issued) $ deposit mirror minAdaTxOut
+              when (issued == Issued) $ deposit mirror (Ada.lovelaceOf 0)
               isIssued w %= doRevoke
             Issue w   -> do
               wait 1
-              withdraw mirror minAdaTxOut
+              withdrawMinted mirror $ someTokenValue "kyc" 1
```

Comment:
In curious here, what is this supposed to do?

Reply:
As `ContractModel` layer introduces it's own state above contracts we have to make them converge.

In `ContractModel` it's possible to set the initial distribution of wallets and add tokens for example. But in this case, in the underlying contract, we issue a token and then withdraw it at the same time. To trigger that in `ContractModel` I added `withdrawMinted` that deposits that token and then immediately withdraws.